lib/src/files.rs

```diff
@@ -330,17 +330,6 @@ mod tests {
             merge(&[b"a", b"a"], &[b"b", b"b", b"b"]),
             MergeResult::Resolved(b"b".to_vec())
         );
-        // Two sides left one line unchanged, and added conflicting additional lines
-        assert_eq!(
-            merge(&[b"a\n"], &[b"a\nb\n", b"a\nc\n"]),
-            MergeResult::Conflict(vec![
-                MergeHunk::Resolved(b"a\n".to_vec()),
-                MergeHunk::Conflict(ConflictHunk {
-                    removes: vec![b"".to_vec()],
-                    adds: vec![b"b\n".to_vec(), b"c\n".to_vec()]
-                })
-            ])
-        );
```

Comment:
I see that this non-trivial case gets moved after the other trivial cases. Makes sense.